src/libmongoc/doc/learn/get/from-source.rst

```diff
@@ -0,0 +1,376 @@
+   - The download is smaller and stored in a single file.
+
+
+.. _learn.obtaining.git:
+
+Downloading Using Git
+=====================
+
+Using Git, the C driver repository can be cloned from the GitHub URL
+``https://github.com/mongodb/mongo-c-driver.git``. Git tags for released
+versions are named after the version for which they correspond (e.g.
+"|version.pre|"). To clone the repository using the command line, the following
+command may be used::
+
+  $ git clone https://github.com/mongodb/mongo-c-driver.git --branch="$VERSION" "$SOURCE_DIR"
+
+
+.. _learn.obtaining.archive:
+
+Downloading a Release Archive
+=============================
+
+An archived snapshot of the repository can be obtained from the
+`GitHub Releases Page`__. The ``mongo-c-driver-x.y.z.tar.gz`` archive attached
+to any release contains a minimal version of the repository with only the files
+necessary to build the driver binaries.
+
+__ https://github.com/mongodb/mongo-c-driver/releases
+
+The archive can be downloaded and extracted from a command line using one of
+several tools, depending on what is available.
+
+.. tab-set::
+
+  .. tab-item:: Using ``wget`` + ``tar``
+
+    ::
+
+      ## Download using wget:
+      $ wget "https://github.com/mongodb/mongo-c-driver/archive/refs/tags/$VERSION.tar.gz" \
+          --output-document="mongo-c-driver-$VERSION.tar.gz"
+      ## Extract using tar:
+      $ tar -x -f "mongo-c-driver-$VERSION.tar.gz"
+
+  .. tab-item:: Using ``curl`` + ``tar``
+
+    ::
+
+      ## Using curl:
+      $ curl "https://github.com/mongodb/mongo-c-driver/archive/refs/tags/$VERSION.tar.gz" \
+          --output="mongo-c-driver-$VERSION.tar.gz"
+      ## Extract using tar:
+      $ tar -x -f "mongo-c-driver-$VERSION.tar.gz"
+
+  .. tab-item:: Using PowerShell
+
+    .. code-block:: pwsh
+
+      ## Use Invoke-WebRequest:
+      Invoke-WebRequest `
+          -UseBasicParsing `
+          -Uri "https://github.com/mongodb/mongo-c-driver/archive/refs/tags/$VERSION.zip" `
+          -OutFile "mongo-c-driver-$VERSION.zip"
+      ## Extract using Expand-Archive:
+      Expand-Archive mongo-c-driver-$VERSION.zip
+
+The above commands will create a new directory `mongo-c-driver-$VERSION` within
+the directory in which you ran the `tar`/`Expand-Archive` command (**note**:
+PowerShell will create an additional intermediate subdirectory of the same
+name). This directory is the root of the driver source tree (which we refer to
+as `$SOURCE` in these documents). The `$SOURCE` directory should contain the
+top-level `CMakeLists.txt` file.
+
+
+Obtaining Prerequisites
+***********************
+
+In order to build the project, a few prerequisites need to be available.
+
+Both |libmongoc| and |libbson| projects use CMake__ for build configuration.
+
+__ https://cmake.org
+
+.. note::
+
+  It is *highly recommended* -- but not *required* -- that you download the
+  latest stable CMake available for your platform.
+
+.. dropdown:: Getting the Latest CMake …
```

Comment:
Other dropdowns have "Aside:" in their title. Do we want one here? (Admittedly I'm of the opinion they're superfluous given the purpose is denoted by the admonition type and would rather they be omitted.)

Reply:
I was searching for a way to visually distinguish dropdown-admonitions from regular admonitions, but I think I can do better by adding an inset using a CSS margin.